cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -119,7 +118,7 @@ public Boolean call() throws Exception {
           throw Throwables.propagate(ex);
```

Comment:
I don't think we should throw a ``ServiceUnavailableException``. That exception is for a service to indicate to clients that the service is unavailable, with an HTTP-like response code. Not sure if it makes sense to throw that from something that's not a service like ``IntegrationTestBase``. In any case, ``ServiceUnavailableException`` is a ``RuntimeException``, so the effect is similar to ``Throwables.propagate(e)``.